runtime/compiler/env/VMJ9Server.hpp

```diff
@@ -292,6 +292,7 @@ class TR_J9SharedCacheServerVM: public TR_J9ServerVM
 
    virtual bool shouldDelayAotLoad() override                                  { return true; }
    virtual bool isStable(int cpIndex, TR_ResolvedMethod *owningMethod, TR::Compilation *comp) override { return false; }
+   virtual bool isForceInline(TR_ResolvedMethod *method) override { return false; }
```

Comment:
This method should be overridden in `TR_J9ServerVM` instead.

Reply:
Right! Fixed.